src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    private String from;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    public String toString() {
+        return String.format("[E][%s] %s (from: %s to: %s)", this.getStatusIcon(),
```

Comment:
This line is confusing and seems to be doing a lot in one line. Perhaps it would be better to break it up or use another way to achieve this result.

Reply:
I'm personally not sure of how to go about this. Perhaps I could use a taskType variable (in the parent class), and move some of the functionality into the parent class?